crawlers/stack-overflow/main.py

```diff
@@ -0,0 +1,17 @@
+import time
+import logging
+
+from stackoverflow.stackoverflow import StackOverflowCrawler
+from stackoverflow.utils.utils import Utils
+
+logging.basicConfig(level=logging.DEBUG)
+
+if __name__ == '__main__':
+    config = Utils.load_json_file('config.json')
+
+    while True:
+        crawler = StackOverflowCrawler(config['keywords'], config['redis']['host'], config['redis']['port'],
```

Comment:
It's not optimal to reconnect every delay. Probably you should better move initialization upper